src/boot/efi/boot.c

```diff
@@ -963,6 +963,7 @@ static bool menu_run(
 
                 case KEYPRESS(EFI_CONTROL_PRESSED, 0, 'l'):
                 case KEYPRESS(EFI_CONTROL_PRESSED, 0, CHAR_CTRL('l')):
+                case 'L': /* only uppercase, do not conflict with lower-case 'l' which picks first Linux entry */
```

Comment:
Not documented??

Reply:
I noticed that, and thought that we already document ctrl-l, so we don't need to document shift-l. But maybe we should, hmm.